src/Components/Color/ColorForm.jsx

```diff
@@ -69,8 +69,11 @@ export default function ColorForm({ addColor }) {
         defaultValue={contrastText}
         onChange={(value) => setContrastText(value)}
       />
-
-      <button type="submit">ADD COLOR</button>
+      {isEditing ? (
+        <button type="submit">UPDATE COLOR</button>
+      ) : (
+        <button type="submit">ADD COLOR</button>
+      )}
```

Comment:
You could actually only render the button text conditionally as the button itself stays the same.
```suggestion
        <button type="submit">{ isEditing ? "UPDATE COLOR" : "ADD COLOR"}</button>
```

Reply:
Great solution, thanks a lot! I will implement this